api/src/certification/session/application/http-error-mapper-configuration.js

```diff
@@ -0,0 +1,26 @@
+import { HttpErrors } from '../../../shared/application/http-errors.js';
+import {
+  SessionWithoutStartedCertificationError,
+  SessionWithAbortReasonOnCompletedCertificationCourseError,
+  SessionAlreadyFinalizedError,
+} from '../domain/errors.js';
+import { DomainErrorMappingConfiguration } from '../../../shared/application/models/domain-error-mapping-configuration.js';
+
+const sessionDomainErrorMappingConfiguration = [
+  {
+    name: SessionWithoutStartedCertificationError.name,
+    httpErrorFn: (error) => {
+      return new HttpErrors.BadRequestError(error.message, error.code, error.meta);
+    },
```

Comment:
Pour faire un peu moins lourd
```suggestion
    httpErrorFn: (error) => (new HttpErrors.BadRequestError(error.message, error.code, error.meta)),
```

Reply:
c'est deja fait comme ca plus bas en fait